chainer/functions/array/vstack.py

```diff
@@ -52,10 +52,11 @@ def vstack(xs):
         xs (list of :class:`~chainer.Variable` or :class:`numpy.ndarray` or \
         :class:`cupy.ndarray`):
             Input variables to be concatenated. The variables must have the
-            same ``ndim``. Also, when the variables have vertical dimension
-            (i.e. :math:`ndim \\geq 1`), the variables must have the same
-            shape, except in the vertical (row wise) dimension.
-
+            same ``ndim``. When the variables have the second axis (i.e.
```

Comment:
When concatenating the 1dim array, the variables must have the same shape along all ~~but the first axis~~... Are there my misunderstanding?

```
>>> import numpy as np
>>> import chainer.functions as F
>>> x1 = np.arange(0, 12)
>>> x2 = np.arange(0, 12)
>>> F.vstack([x1, x2]).data
array([[ 0,  1,  2,  3,  4,  5,  6,  7,  8,  9, 10, 11],
          [ 0,  1,  2,  3,  4,  5,  6,  7,  8,  9, 10, 11]])
>>> x2 = np.arange(0, 11)
>>> F.vstack([x1, x2]).data
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/kumezawa/dev/chainer/chainer/functions/array/vstack.py", line 59, in vstack
    return Vstack()(*xs)
  File "/home/kumezawa/dev/chainer/chainer/function.py", line 189, in __call__
    self._check_data_type_forward(in_data)
  File "/home/kumezawa/dev/chainer/chainer/function.py", line 273, in _check_data_type_forward
    self.check_type_forward(in_type)
  File "/home/kumezawa/dev/chainer/chainer/functions/array/vstack.py", line 23, in check_type_forward
    type_check.expect(in_types[0].shape == in_types[i].shape)
  File "/home/kumezawa/dev/chainer/chainer/utils/type_check.py", line 487, in expect
    expr.expect()
  File "/home/kumezawa/dev/chainer/chainer/utils/type_check.py", line 449, in expect
    '{0} {1} {2}'.format(left, self.inv, right))
chainer.utils.type_check.InvalidType: 
Invalid operation is performed in: Vstack (Forward)

Expect: in_types[0].shape == in_types[1].shape
Actual: (12,) != (11,)
```